compiler/prelude/prelude_min.go

```diff
@@ -0,0 +1,6 @@
e(e,t.elem)});case $kindFunc:return function(){var args=[];for(var i=0;i<t.params.length;i++){if(t.variadic&&i===t.params.length-1){var vt=t.params[i].elem,varargs=arguments[i];for(var j=0;j<varargs.$length;j++){args.push($externalize(varargs.$array[varargs.$offset+j],vt))}break}args.push($externalize(arguments[i],t.params[i]))}var result=v.apply(recv,args);switch(t.results.length){case 0:return;case 1:return $internalize(result,t.results[0]);default:for(var i=0;i<t.results.length;i++){result[i]=$internalize(result[i],t.results[i])}return result}};case $kindInterface:if(t.methods.length!==0){$throwRuntimeError("cannot internalize "+t.string)}if(v===null){return $ifaceNil}if(v===undefined){return new $jsObjectPtr(undefined)}switch(v.constructor){case Int8Array:return new($sliceType($Int8))(v);case Int16Array:return new($sliceType($Int16))(v);case Int32Array:return new($sliceType($Int))(v);case Uint8Array:return new($sliceType($Uint8))(v);case Uint16Array:return new($sliceType($Uint16))(v);case Uint32Array:return new($sliceType($Uint))(v);case Float32Array:return new($sliceType($Float32))(v);case Float64Array:return new($sliceType($Float64))(v);case Array:return $internalize(v,$sliceType($emptyInterface));case Boolean:return new $Bool(!!v);case Date:if(timePkg===undefined){return new $jsObjectPtr(v)}return new timePkg.Time($internalize(v,timePkg.Time));case Function:var funcType=$funcType([$sliceType($emptyInterface)],[$jsObjectPtr],true);return new funcType($internalize(v,funcType));case Number:return new $Float64(parseFloat(v));case String:return new $String($internalize(v,$String));default:if($global.Node&&v instanceof $global.Node){return new $jsObjectPtr(v)}var mapType=$mapType($String,$emptyInterface);return new mapType($internalize(v,mapType))}case $kindMap:var m={};var keys=$keys(v);for(var i=0;i<keys.length;i++){var k=$internalize(keys[i],t.key);m[t.key.keyFor(k)]={k:k,v:$internalize(v[keys[i]],t.elem)}}return m;case $kindPtr:if(t.elem.kind===$kindStruct){return $internalize(v,t.elem)}case $kindSlice:return new t($mapArray(v,function(e){return $internalize(e,t.elem)}));case $kindString:v=String(v);if($isASCII(v)){return v}var s="";var i=0;while(i<v.length){var h=v.charCodeAt(i);if(55296<=h&&h<=56319){var l=v.charCodeAt(i+1);var c=(h-55296)*1024+l-56320+65536;s+=$encodeRune(c);i+=2;continue}s+=$encodeRune(h);i++}return s;case $kindStruct:var noJsObject={};var searchJsObject=function(t){if(t===$jsObjectPtr){return v}if(t===$jsObjectPtr.elem){$throwRuntimeError("cannot internalize js.Object, use *js.Object instead")}switch(t.kind){case $kindPtr:return searchJsObject(t.elem);case $kindStruct:var f=t.fields[0];var o=searchJsObject(f.typ);if(o!==noJsObject){var n=new t.ptr;n[f.prop]=o;return n}return noJsObject;default:return noJsObject}};var o=searchJsObject(t);if(o!==noJsObject){return o}}$throwRuntimeError("cannot internalize "+t.string)};var $isASCII=function(s){for(var i=0;i<s.length;i++){if(s.charCodeAt(i)>=128){return false}}return true};
```

Comment:
`Minified` was 32740 bytes before.

Now it's 41046 bytes. How come? Can we make it match previous output (from https://skalman.github.io/UglifyJS-online/ with its default options) or closer to it in size?

Reply:
Thanks, good catch. So it appears Actually it appears https://skalman.github.io/UglifyJS-online/ is using some non-default options. I've added `uglify_options.json` in a [new commit](https://github.com/gopherjs/gopherjs/pull/784/commits/5633de29373c2e6456ca8c74313b09785d39524f) and now things look better:

```
$ git rev-parse HEAD
5633de29373c2e6456ca8c74313b09785d39524f
$ cat compiler/prelude/prelude_min.go | wc -c
33300
```